modules/circeDerivation/src/main/scala/evo/derivation/circe/EvoDecoder.scala

```diff
@@ -29,7 +29,7 @@ object EvoDecoder:
         summonFrom {
             case given Mirror.ProductOf[A] => deriveForProduct[A].instance
             case given Mirror.SumOf[A]     => deriveForSum[A]
-            case given ValueClass[A]       => deriveForNewtype[A]
+            case given ValueClass[A]       => deriveForValueClass[A]
```

Comment:
I don't like this renaming, since I'm going to use this branch for more than value classes supposedly

Reply:
Feel free to revert it then later